### ISSUE TITLE
video background

### ISSUE THREAD

**Issue by jor1196** [ubports/morph-browser#37]
that comes with the feature of playing videos and not pause when you lock the device screen, changing the browser tab / window or using another application

**Comment by mateosalta** [comment]
yep, I agree media-hub usage would be handy, maybe a toggle - down on the new tab pull up?

**Comment by jor1196** [comment]
> yep, I agree media-hub usage would be handy, maybe a toggle - down on the new tab pull up?
this will help?

https://github.com/brave/browser-android-tabs/issues/744

**Comment by Fuseteam** [comment]
+1 for mediahub integration
adding the link mateo found for a possible fix here:
https://bugs.launchpad.net/canonical-devices-system-image/+bug/1603729

**Comment by mateosalta** [comment]
that was only for the old oxide browser, but might provide insight if you could track down how it was done for audio previously.
